packages/client/src/checkout/patchCheckoutOrderDeliveryBundles.ts

```diff
@@ -0,0 +1,47 @@
+import { adaptError } from '../helpers/client/formatError.js';
+import client from '../helpers/client/index.js';
+import type { PatchCheckoutOrderDeliveryBundles } from './types/index.js';
+
+/**
+ * Method responsible for applying the selected delivery bundle upgrade for the
```

Comment:
Is this description correct? It seems the same for the patchCheckoutOrderDeliveryBundleUpgrades client